modules/core/src/smithy4s/schema/EnumTag.scala

```diff
@@ -0,0 +1,8 @@
+package smithy4s.schema
+
+sealed trait EnumTag
```

Comment:
**suggestion:** I'd move this to live next to `EnumerationSchema` rather than in a new file

Reply:
I disagree. I'd rather it stayed in its own file.